fe/fe-core/src/main/java/org/apache/doris/nereids/glue/translator/PhysicalPlanTranslator.java

```diff
@@ -434,28 +432,65 @@ public PlanFragment visitPhysicalProject(PhysicalProject<Plan> project, PlanTran
                 .map(e -> ExpressionTranslator.translate(e, context))
                 .collect(Collectors.toList());
         // TODO: fix the project alias of an aliased relation.
+        List<Slot> slotList = project.getOutput();
```

Comment:
To a common format, we do not use List, Set, ... as suffix usually, instead we use plural, i.e. slots. Please fix other place in this file, thanks

Reply:
I don't find the explicit specification for varibale name style of Java code in the developer documentation. Is there any benefits to do this?

Reply:
List in name is redundant with its type, and it is longer than just use plural

Reply:
But when read a complex function with many lines, collections types in the variable name would be a hint for some characterics of the objective.